src/routes/+page.svelte

```diff
@@ -1,2 +1,40 @@
-<h1>Welcome to SvelteKit</h1>
-<p>Visit <a href="https://kit.svelte.dev">kit.svelte.dev</a> to read the documentation</p>
+<h1>Welcome to Kendo Club!</h1>
+
+<h2>What is Kendo?</h2>
+<p>
+	Kendo is a Japanese martial art which literally translates to “The Way of the Sword”. It's an
+	athletic sport which is played by means of one-on-one striking between opponents using shinai
+	(sword made from bamboo slats) while wearing kendo-gu (protective armor).
+</p>
+
+<h2>Who are we?</h2>
+
+Kendo Club at the University of Michigan is led by students and instructed by highly experienced
+coaches. Our purpose is to provide an open and inviting opportunity for University of Michigan
+students and local community members to learn and practice Kendo. We invite everyone of any skill
+level to join us!
+
+<h2>How do you join?</h2>
+
+<h3>Come to practice!</h3>
+<ul>
+	<li>
+		You may join any practice session you like (though you may find it easier to join closer to the
+		beginning of a new semester, as there are larger groups of complete beginners).
+	</li>
+	<li>Find our up-to-date practice schedule in the <a href="/calendar">Calendar</a> page.</li>
+	<li>
+		Please wear comfortable athletic clothes. Wear bottoms that cover the knee because there is some
+		kneeling involved. Kendo is a barefoot sport, so please leave your shoes outside the ballroom
+		before entering.
+	</li>
+	<li>Masking is optional. Surgical or cloth masks are the best choices for breathability.</li>
+</ul>
+
+<h3>Email us</h3>
+<ul>
+	<li>
+		You can find our email address in the <a href="/contact-us">Contact Info</a> page. We can answer
```

Comment:
It feels a little bit awkward to link to the contact page for the email but I guess it's alright. It makes me wonder if the contact info should just be at the bottom of this page.

Reply:
We should discuss that at the stakeholder meeting. I'll add it to #22